demo/main.js

```diff
@@ -189,6 +189,12 @@ class Demo extends Component {
             inputFiles[images[i]] = { imageSrc };
           }
           await this.initiateConnection(inputFiles);
+          stores.resetStores();
+          stores.inject(NotificationStore, {
+            notifyUser: this.notifyUser,
+            raiseError: this.raiseError,
+            askConfirmation: this.askConfirmation,
+          });
```

Comment:
Why is it necessary ? It's seems to not be related to combo chart and it's rather a fix ? I would put it in a dedicated commit

Reply:
This part of the code is from a PR of Adrien, so I just added it to my code to use it for testing the export part